migrations/20230427070145-add-Agreements-table.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('Agreements', {
+      id: {
+        type: Sequelize.INTEGER,
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+      },
+      title: {
+        type: Sequelize.STRING,
+        allowNull: false,
+      },
+      expiresAt: {
+        allowNull: true,
+        type: Sequelize.DATE,
+      },
+      UserId: {
+        type: Sequelize.INTEGER,
+        references: { key: 'id', model: 'Users' },
+        allowNull: true,
+        onDelete: 'CASCADE',
+        onUpdate: 'CASCADE',
+      },
+      HostCollectiveId: {
+        type: Sequelize.INTEGER,
+        references: { key: 'id', model: 'Collectives' },
+        allowNull: true,
+        onDelete: 'CASCADE',
+        onUpdate: 'CASCADE',
+      },
+      CollectiveId: {
+        type: Sequelize.INTEGER,
+        references: { key: 'id', model: 'Collectives' },
+        allowNull: true,
+        onDelete: 'CASCADE',
+        onUpdate: 'CASCADE',
+      },
+      UploadedFileId: {
+        type: Sequelize.INTEGER,
+        references: { key: 'id', model: 'UploadedFiles' },
+        allowNull: true,
+        onDelete: 'CASCADE',
+        onUpdate: 'CASCADE',
+      },
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE,
+        defaultValue: Sequelize.fn('now'),
+      },
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE,
+        defaultValue: Sequelize.fn('now'),
+      },
+      deletedAt: {
+        allowNull: true,
+        type: Sequelize.DATE,
```

Comment:
Maybe we could add `archivedAt` to enable archiving

Reply:
Since it will be an nullable field, can we add it when we enable archiving?

Reply:
Sounds good!